apiv2/db_import/tests/populate_db.py

```diff
@@ -438,6 +464,24 @@ def populate_stale_tiltseries(session: sa.orm.Session) -> None:
     )
 
 
+@write_data
+def populate_stale_per_section_parameters(session: sa.orm.Session) -> PerSectionParameters:
+    session.add(
+        PerSectionParameters(
```

Comment:
Could we also add a testing for the case where we are deleting a record that was added in the past with incorrect data for a valid run, and the updated data no longer contains that record?